insights/specs/__init__.py

```diff
@@ -278,8 +278,6 @@ class Specs(SpecSet):
     ksmstate = RegistryPoint()
     kubepods_cpu_quota = RegistryPoint(multi_output=True)
     lastupload = RegistryPoint(multi_output=True)
-    libkeyutils_objdumps = RegistryPoint()
-    libkeyutils = RegistryPoint()
```

Comment:
@matysek don't forget to remove `woopsie` as well from this file.

Reply:
Yeah, the variable was called 'woopsie' and not 'whoopsie' so forgot to remove it. Thank you!.